src/stories/Foundation/Tooltip/index.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import { Text } from '../Text';
+import { Flex } from '../Flex';
+
+const Wrapper = styled.div`
+  position: relative;
+  width: fit-content;
+  &:hover {
+    div {
+      display: flex;
+    }
+  }
+`;
+
+const ToolTips = styled(Flex)`
+  display: none;
+  position: absolute;
+  // top and left is relative to size of wrapper
+  left: 50%;
+  top: -4px;
```

Comment:
Avoid negative values for positioning. I would do this instead to avoid the tooltip disappearing if you hover over to the tooltip content too slow:
```css
top: 0%;
transform: translate(-50%, -104%);
```
Transform doesn't really reposition things technically, so it wouldn't affect hovering iirc.

Also I guess positioning it at the top always is fine for now, but positioning should prob be determined by the viewport in the future in case a user scrolls the tooltip out of window.